modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/analysis-common/20_analyzers.yml

```diff
@@ -957,7 +957,7 @@
             text:     absenţa
             analyzer: my_analyzer
     - length: { tokens: 1 }
-    - match:  { tokens.0.token: absenţ }
+    - match:  { tokens.0.token: absenț }
```

Comment:
@cbuescher I assume that these tests don't run in a mixed cluster , because they are in analysis-common, hence not part of the ordinary yaml tests? Otherwise, I think that a cluster feature should have been used in here, because this test would fail in a mixed cluster? This is different from the rest compat stuff that you addressed in the build file by replacing tokens.

Reply:
I think so, yes. But I haven't checked it, I assume something would fail on the lucene_snapshot branch otherwise